### PR TITLE
Cut release 0.29.0

### PR DESCRIPTION
**Pull request by balopat**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,65 @@
 # Minikube Release Notes
 
+# Version 0.29.0 - 09/27/2018
+* Issue #3037 change dependency management to dep [#3136](https://github.com/kubernetes/minikube/pull/3136)
+* Update dashboard version to v1.10.0 [#3122](https://github.com/kubernetes/minikube/pull/3122)
+* fix: --format outputs any string, --https only subsitute http URL scheme [#3114](https://github.com/kubernetes/minikube/pull/3114)
+* Change default docker storage driver to overlay2 [#3121](https://github.com/kubernetes/minikube/pull/3121)
+* Add env variable for default ES_JAVA_OPTS [#3086](https://github.com/kubernetes/minikube/pull/3086)
+* fix(cli): `minikube start --mount --mountsting` without write permission [#2671](https://github.com/kubernetes/minikube/pull/2671)
+* Add workaround for dnsmasq and hyperkit users to drivers readme [#3003](https://github.com/kubernetes/minikube/pull/3003)
```
Comment (tstromberg) [inline CHANGELOG.md]: This is just a README update, and not related to the release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,65 @@
...
+* Allow certificates to be optionally embedded in .kube/config [#3065](https://github.com/kubernetes/minikube/pull/3065)
+* Fix the --cache-images flag. [#3090](https://github.com/kubernetes/minikube/pull/3090)
+* support containerd  [#3040](https://github.com/kubernetes/minikube/pull/3040)
+* Fix vmwarefusion driver [#3029](https://github.com/kubernetes/minikube/pull/3029)
+* Make CoreDNS default addon [#3072](https://github.com/kubernetes/minikube/pull/3072)
+* Update CoreDNS deployment [#3073](https://github.com/kubernetes/minikube/pull/3073)
+* readme: update the script to match executable path [#3051](https://github.com/kubernetes/minikube/pull/3051)
```
Comment (tstromberg) [inline CHANGELOG.md]: This too.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,65 @@
...
+- Daemeron
+- Damian Kubaczka
+- Daniel Santana
+- dlorenc
+- Jason Stangroome
+- Jeffrey Sica
+- Joao Carlos
+- Kumbirai Tanekha
+- Matt Rickard
+- minikube-bot
```
Comment (tstromberg) [inline CHANGELOG.md]: Maybe not?

File: pkg/minikube/cluster/cluster.go [hunk 4]
```diff
@@ -217,7 +217,7 @@ func createHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error
 
 	if config.VMDriver != "none" {
 		if err := config.Downloader.CacheMinikubeISOFromURL(config.MinikubeISO); err != nil {
-			return nil, errors.Wrap(err, "Error attempting to cache minikube ISO from URL")
+			return nil, fmt.Errorf("error attempting to cache minikube ISO from URL (%s): %s", config.MinikubeISO, err)
```
Comment (tstromberg) [inline pkg/minikube/cluster/cluster.go]: Unrelated fix?

File: test/integration/cluster_status_test.go [hunk 5]
```diff
@@ -47,7 +47,7 @@ func testClusterStatus(t *testing.T) {
 			}
 			if status != api.ConditionTrue {
 				err := fmt.Errorf("Component %s is not Healthy! Status: %s", i.GetName(), status)
-				t.Log("Retrying, %s", err)
+				t.Logf("Retrying, %s", err)
```
Comment (tstromberg) [inline test/integration/cluster_status_test.go]: Unrelated fix here too.

File: CHANGELOG.md (hunk 3 above)
Comment (balopat) [reply CHANGELOG.md]: lol

File: pkg/minikube/cluster/cluster.go (hunk 4 above)
Comment (balopat) [reply pkg/minikube/cluster/cluster.go]: okay...I'll remove them...
